### PR TITLE
Updates the custom-tabular-bq-managed-dataset notebook according to the new template

### PR DESCRIPTION
**Pull request by krishr2d2**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 1]
```diff
@@ -122,7 +122,7 @@
       "source": [
```
Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>Add BigQuery to pricing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>combine installs to single cell</p><p>Add -q option</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>use template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>Add text cell</p><p> </p><p>The import should be moved up into the prev cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>Add text cell</p><p> </p><p>Move import into the import cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>combine with code cell below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <h2>managed tabular dataset -&gt; Vertex AI Tabular Dataset</h2><p> </p><p>Add explain of call and params</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:122]: <p>Each param on its own line. Ex</p><p> </p><p>result = aiplatform.method(</p><p>    param1=...,</p><p>    paramN=...</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: replaced master with main...

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>Added the used resources section</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>added...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>combined....</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>added and updated....</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>added and updated...</p>

Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>done...</p>

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:131]: <p>updated....</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**